app/lib/package/backend.dart

```diff
@@ -70,17 +62,13 @@ PackageBackend get packageBackend =>
 /// Represents the backend for the pub site.
 class PackageBackend {
   final DatastoreDB db;
-  final TarballStorage _storage;
-  final int _maxVersionsPerPackage;
-
-  PackageBackend(
-    DatastoreDB db,
-    TarballStorage storage, {
-    int? maxVersionsPerPackageOverride,
-  })  : db = db,
-        _storage = storage,
-        _maxVersionsPerPackage =
-            maxVersionsPerPackageOverride ?? maxVersionsPerPackage;
+  final Bucket _bucket;
```

Comment:
Let's give this a documentation string, stating the layout of what is stored in here..